numpy/_core/tests/test_scalarprint.py

```diff
@@ -15,13 +15,13 @@ def test_str(self):
         svals = [0.0, -0.0, 1, -1, np.inf, -np.inf, np.nan]
         styps = [np.float16, np.float32, np.float64, np.longdouble]
         wanted = [
-             ['0.0',  '0.0',  '0.0',  '0.0' ],
+             ['0.0',  '0.0',  '0.0',  '0.0' ],  # noqa: E202
```

Comment:
Curious about E202. I'm guessing alignment?

Reply:
[E202](https://docs.astral.sh/ruff/rules/whitespace-before-close-bracket/) checks for the use of extraneous whitespace before `)`, `]` or `}`.